packages/clarity-js/types/data.d.ts

```diff
@@ -69,7 +69,8 @@ export const enum Metric {
     ColorDepth = 16,
     ReviewCount = 17,
     BestRating = 18,
-    WorstRating = 19
+    WorstRating = 19,
+    CartTotal = 20
```

Comment:
Since these are driven by configuration should they not be declared here? Can we, instead, reserve a range of value here that configuration is limited to?

Reply:
We can technically override any metric value. For instance even ProductPrice or others similar to regions. We will add more checks as we get better understanding of it. 

---
In reply to: [595644826](https://github.com/microsoft/clarity/pull/121#discussion_r595644826) [](ancestors = 595644826)